src/Data/Integer/Properties.agda

```diff
@@ -199,6 +209,8 @@ drop‿-<- (-<- n<m) = n<m
 ≰⇒> { -[1+_] n} {+_ n₁}      i≰j = contradiction -≤+ i≰j
 ≰⇒> { -[1+_] n} { -[1+_] n₁} i≰j = -<- (ℕₚ.≰⇒> (i≰j ∘ -≤-))
 
+≯⇒≤ : ∀ {x y} → x ≯ y → x ≤ y
+
```

Comment:
I understand the rationale for putting the type signature in this block
however this generate an unnecessary mutual block (and to the reader
not looking at a diff, it looks like a declaration with no definition).

Reply:
That's very true and bothered me even while writing this down, anyways I botched the proof, I rewrote it and now definitions are gathered and all clean.